examples/plotting/file/linked_brushing.py

```diff
@@ -1,3 +1,11 @@
+'''Two scatter plots one representing # Cylinders vs. MPG and the second graph representing acceleration vs. MPG
```

Comment:
This leaves out the best part about the linked brushing! Take it for a test run with `python examples/plotting/file/linked_brushing.py`